src/schrodinger/sketcher/sketcher_widget.cpp

```diff
@@ -493,10 +493,11 @@ void SketcherWidget::connectTopBarSlots()
             &QUndoStack::undo);
     connect(m_ui->top_bar_wdg, &SketcherTopBar::redoRequested, m_undo_stack,
             &QUndoStack::redo);
-    connect(m_ui->top_bar_wdg, &SketcherTopBar::cleanupRequested, [this]() {
-        m_mol_model->regenerateCoordinates();
-        m_ui->view->fitToScreen();
-    });
+    connect(m_ui->top_bar_wdg, &SketcherTopBar::cleanupRequested,
+            [this](bool selection_only) {
+                m_mol_model->regenerateCoordinates(selection_only);
```

Comment:
I think it might be a little clearer to have something like
```cpp
if (selection_only) {
    m_mol_model->cleanUpSelection();
} else {
    m_mol_model->regenerateCoordinates();
}
```
here and get rid of the `selection_only` parameter to `regenerateCoordinates`.  It seems a little weird for `regenerateCoordinates` to have a parameter that causes it to just call a different method instead of itself.